docs/web/permissions.md

```diff
@@ -165,3 +166,11 @@ status, etc.
 
 Users need the `PRODUCT_STORE` permission to `store` analysis runs and to
 delete existing analysis runs from the server.
+
+### `PRODUCT_VIEW` <a name="product-view"></a>
```

Comment:
Now If I login with a user who has only this permission the GUI will show me that the product is not accessible. For this you have to change this line: https://github.com/Ericsson/codechecker/blob/53fc3be8619e011e13ce0dd41b6f58fc826bf336/web/server/codechecker_server/api/product_server.py#L117
Change PRODUCT_ACCESS to PRODUCT_VIEW.

Reply:
Done!